code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -28,6 +30,11 @@
 		var/datum/role/traitor/newTraitor = new
 		newTraitor.AssignToRole(M.mind,1)
 		newTraitor.Greet(GREET_ROUNDSTART)
+		// Above 3 traitors, we start to cost a bit more.
+		if (i > max_traitors && (mode.threat_level > additional_cost))
```

Comment:
Threat level 5 (out of 100) is basically extended already, I don't know the purpose of having this check here, ditto for vampires

Reply:
I was certain `threat_level`  was the remaining threat. It's actually `threat`, my mistake.